### PR TITLE
Retain single source of truth for Web Chat Licensing information in PC & Console section

### PR DESCRIPTION
**Pull request by hs-saunved**

### CODE REVIEW

**Review by helpshift-ps — APPROVED** [review]

File: docs/pc-and-console/licensing.mdx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 sidebar_position: 80
 title: License Information
-description: "Support Experience Link and QR Code linked Webchat uses following third party Javascript libraries:"
+description: "Third-party libraries used by QR experience linked Web Chat"
```
Comment (collinx) [inline docs/pc-and-console/licensing.mdx:4]: This should be

> Third-party libraries used by QR Code experience linked Web Chat

File: docs/pc-and-console/licensing.mdx [hunk 2]
```diff
@@ -23,84 +23,8 @@ import {
 
 <Intro>
 
-QR experience linked Webchat uses following third party Javascript libraries:
+Third-party libraries used by QR experience linked Web Chat
```
Comment (collinx) [inline docs/pc-and-console/licensing.mdx:26]: This should be
> Third-party libraries used by QR Code experience linked Web Chat

File: docs/pc-and-console/licensing.mdx [hunk 3]
```diff
@@ -23,84 +23,8 @@ import {
...
-
-|         |                                                                                  |
-| ------- | -------------------------------------------------------------------------------- |
-| Version | v2.2.0                                                                           |
-| Purpose | Redux middleware to dispatch async actions                                       |
-| Source  | [https://github.com/gaearon/redux-thunk](https://github.com/gaearon/redux-thunk) |
-| License | MIT                                                                              |
-
-</div>
+Please refer to the <a href="/web-chat/licensing" target="_blank">Web Chat Licensing Information</a> for the third-party Javascript libraries used by QR experience linked Web Chat.
```
Comment (collinx) [inline docs/pc-and-console/licensing.mdx:30]: Same for this

**Review by collinx — COMMENTED** [review]
We should use QR Code experience or Support Experience Link and QR Code.

**Review by helpshift-vishal — APPROVED** [review]

**Pull request merged**